lib/modules/apostrophe-express/index.js

```diff
@@ -265,7 +267,7 @@ module.exports = {
             if (!url.match(/^[a-zA-Z]+:/)) {
               url = self.apos.prefix + url;
             }
-            return superRedirect.call(this, status, url);
+            return res.rawRedirect.call(res, status, url);
```

Comment:
It looks like this no longer needs to use `.call`. `res.rawRedirect(status, url)`. eslint is throwing on that.